app/static/js/createEvents.js

```diff
@@ -168,5 +169,23 @@ $(document).ready(function() {
      $(this).closest("tr").remove();
   });
 
+  $(".removeAttachment").on("click", function(){
 
+    let fileId=  $(this).data("id")
+      let fileData = {fileId : fileId,
+                      eventId:this.id}
+      $("#attachment_"+fileId).remove()
```

Comment:
I wouldn't remove this until the ajax request was successful. If it fails for some reason, we don't want that row to disappear